Sources/AardvarkRevealTests/ApplicationStateTests.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  Copyright 2020 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+
+@testable import AardvarkReveal
+
+final class ApplicationStateTests: XCTestCase {
+
+    func testClass() throws {
```

Comment:
love these test, but naming the test methods in a more elucidating way would be great. Consider writing test methods using the form `test_thingBeingTested_stateExpectation` to make long-term maintenance easier.